services/QuillLMS/app/controllers/api/v1/activities_controller.rb

```diff
@@ -81,6 +83,16 @@ def published_edition
     render json: {}
   end
 
+  def question_health
+    questions = @activity.data["questions"]
+    tool = CLASSIFICATION_TO_TOOL[ActivityClassification.find(@activity.activity_classification_id).key.to_sym]
+    questions_arr = questions.each.with_index(1).map do |q, question_number|
+      question = Question.find_by(uid: q["key"])
+      question_obj = (question.present? ? QuestionHealthObj.new(@activity, question, question_number, tool).run : {})
```

Comment:
No need to set a temp variable since it isn't used:

`question ? QuestionHealthObj.new(@activity, question, question_number, tool).run : {}`